bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ContainerDataModel.java

```diff
@@ -29,6 +31,7 @@
      *
      * @since com.adobe.cq.wcm.core.components.models.datalayer 1.0.0
      */
+    @JsonProperty("xdm:items")
```

Comment:
IIUC `xdm:items` represents all the items of the container which is different from the displayed items.

Reply:
You're right, we only list the active items. I will switch this back to our custom property name.

Reply:
Done in f71cfef